test/e2e/operator/main_test.go

```diff
@@ -33,7 +34,11 @@ var _ = BeforeSuite(func() {
 
 	err := framework.AddToFrameworkScheme(apis.AddToScheme, nmstateList)
 	Expect(err).ToNot(HaveOccurred())
-	prepare(t)
+	ctx, _ = prepare(t)
+})
+
+var _ = AfterSuite(func() {
```

Comment:
There is something that we miss at e2e test, It's not related to this PR but can you check if "Ctrl-C" get catch by ginkgo test framework ? if it's not can you do some investigation why ?

Reply:
It appears to... 

^Cmake: *** [Makefile:144: test-e2e-operator] Interrupt                                                               
                                                                                                                      
---------------------------------------------------------                                                             
Received interrupt.  Running AfterSuite...                                                                            
^C again to terminate immediately                                                                                     
--- FAIL: TestE2E (3.83s)